code/components/lovely-script/src/Lovely.cpp

```diff
@@ -1,177 +1,173 @@
-/*

```

Comment:
Why does this remove and re-add the entire file?

Reply:
Looks like the PR was broken due to the time it was waiting for, I'll do it again and it should look good